src/data/dialogmote/dialogmoteQueryHooks.ts

```diff
@@ -11,6 +11,11 @@ import { isAktivtDialogmote } from "@/utils/dialogmoteUtils";
 
 export const dialogmoterQueryKeys = {
   dialogmoter: (fnr: string) => ["dialogmoter", fnr],
+  statusendringHistorikk: (fnr: string) => [
+    "dialogmoter",
+    fnr,
+    "statusendringHistorikk",
+  ],
```

Comment:
Ved å bruke samme prefix her på de to querykeyene `["dialogmoter, fnr]`, så vil man invalidere begge når man invaliderer `[dialogmoter, fnr]` ved oppdateringer 👍🏼